contracts/token/ERC20_base.cairo

```diff
@@ -4,152 +4,128 @@ from starkware.cairo.common.cairo_builtins import HashBuiltin, SignatureBuiltin
 from starkware.starknet.common.syscalls import get_caller_address
```

Comment:
I would suggest we use 

`from openzeppelin.token.erc20.library import ERC20`

for these functions